launch/dynamo-run/src/opt.rs

```diff
@@ -88,6 +88,19 @@ impl Default for Input {
     }
 }
 
+impl Input {
+    pub fn available_inputs() -> Vec<String> {
```

Comment:
Is this used anywhere? I don't think we need it, they are all always included, so `--help` will tell them (see `main.rs` at the top)